File: BECore/app/model/WageCoefficient.js
/// <summary>
/// Model EXTJS WageCoefficient.
/// </summary>
Ext.define('QLDT.model.WageCoefficient', {
    extend: 'Ext.data.Model',
    idProperty: 'WageCoefficientID',
    idType: 'Guid',
    fields: [
		{ name: "WageCoefficientID", type: "string" },
		{ name: "GrantRatioID", type: "string" },
		{ name: "ProjectPositionID", type: "string" },
        { name: "ProjectPositionName", type: "string" },
		{ name: "Month", type: "int" },
		{ name: "Year", type: "int" },
		{ name: "Coefficient", type: "float" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: "Edit", type: "boolean" },
    ]
});

File: MT.Model/Business/WageCoefficient.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region WageCoefficient:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a WageCoefficient:EntityBase.
    /// </summary>
    public class WageCoefficient : EntityBase
    {
        [Key]
        public Guid WageCoefficientID { get; set; }
        public Guid GrantRatioID { get; set; }
        public Guid ProjectPositionID { get; set; }
        public String ProjectPositionName { get; set; }
        public int Month { get; set; }
        public int Year { get; set; }
        public decimal Coefficient { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
        /// <summary>
        /// Edit = 1 cho phep nhap
        /// </summary>
        public bool Edit { get; set; }
        
    }
    #endregion
}

